package.json

```diff
@@ -35,7 +35,7 @@
     "release:validate": "./scripts/validateRelease.sh",
     "db:init": "FORCE_DROP=true node ./scripts/sync_models",
     "db:init:memeo:inmem": "FORCE_DROP=true node ./scripts/sync_models && sequelize db:seed --seed memeolist-example-inmem.js",
-    "db:init:memeo:postgres": "FORCE_DROP=true node ./scripts/sync_models && sequelize db:seed --seed memeolist-example-postgres.js && docker exec datasyncserver_postgres_memeo_1 psql -U postgres -d memeolist_db -f /tmp/examples/memeolist.tables.sql",
+    "db:init:memeo:postgres": "FORCE_DROP=true node ./scripts/sync_models && sequelize db:seed --seed memeolist-example-postgres.js && docker exec aerogeardatasyncserver_postgres_memeo_1 psql -U postgres -d memeolist_db -f /tmp/examples/memeolist.tables.sql",
```

Comment:
Okay we need to clear this one up. The command to run this stuff should be `docker-compose up` with no additional options. This will result in containers called `datasyncserver_postgres_memeo_1` and `datasyncserver_postgres_1`. These are the names we should use in these npm commands.

How it works is docker-compose takes the folder name. So if the folder on your local machine is called 'data-sync-server' this will automatically work. The confusion comes from the fact the repo used to be called aerogear-data-sync-server so if you still have it named that way locally you get some issues because the container names will be like this: `aerogeardatasyncserver_postgres_memeo_1`

If your local repo is named `aerogear-data-sync-server` then you need to rename it locally to `data-sync-server`.

Reply:
Let's ensure all of these commands are using the following names.

* `datasyncserver_postgres_memeo_1`
* `datasyncserver_postgres_1`

Then in the Readme we are going to show something like this:

### Getting Started

Start the databases as follows:

```
docker-compose up
```

**Note:** if your local project directory is not named `data-sync-server` you must use `docker-compose up -p datasyncserver`

Reply:
@darahayes  When I run docker compose up I get `Attaching to aerogeardatasyncserver_postgres_memeo_1, aerogeardatasyncserver_postgres_1`

And then the ` npm run db:init:memeo:postgres` fails as it looks for `datasyncserver_postgres_memeo_1 ` Thats why I changed it

Reply:
Is that because my project is called aerogear-data-sync-server?

Reply:
@darahayes , never mind I really should have read the comment above your latest one!

Reply:
@cfoskin `docker-compose` creates container names differently in older versions. For example my version (`1.22.0`) doesn't doesn't replace the dashes in the folder name and it creates container with hame `data-sync-server_<db_name>`. To resolve this problem for now, after call with @darahayes we've decided to change the container names in `package.json` file to `sync_<db-name>` and specify `docker-compose up -p sync` as a default command in readme.